spec/integration/bulkLoad.spec.js

```diff
@@ -0,0 +1,104 @@
+const { config } = testHelpers;
+
+const skwell = require( "src/" );
+describe( "Bulk Load - Integration", () => {
+	let sql;
+	before( async () => {
+		sql = await skwell.connect( config );
+		return sql.execute( sql.fromFile( "sql/bulkLoad-setup.sql" ) );
+	} );
+
+	afterEach( async () => {
+		await sql.execute( "DELETE FROM BulkLoadTest" );
+		await sql.execute( "DROP TABLE IF EXISTS BulkLoadTestNew" );
+	} );
+
+	after( () => {
+		return sql.dispose();
+	} );
+	describe( "connection pool", () => {
+		it( "should not bulk load a temp table", () => {
+			return sql.bulkLoad( "#bulkloadnope", {
+				schema: {
+					id: sql.int
+				},
+				rows: [ { id: 1 } ]
+			} ).should.eventually.be.rejectedWith( Error, "Unable to load temp table '#bulkloadnope' using connection pool. Use a transaction instead." );
+		} );
+
+		it( "should bulk load a table", async () => {
+			await sql.bulkLoad( "BulkLoadTest", {
+				schema: {
+					id: sql.int.nullable()
+				},
+				rows: [ { id: 1 }, { id: 2 }, { id: 3 } ]
```

Comment:
Since `id` is nullable, how about adding a null one?

Reply:
Added a similar test below this one.